packages/near-api-js/src/account_multisig.ts

```diff
@@ -381,6 +396,61 @@ export class Account2FA extends AccountMultisig {
         });
     }
 
+    /**
+     * Returns `false` if multisig cannot be disabled by calling disable() directly
+     * i.e. account has too many LAKs to delete and re-create as FAKs for a single transaction
+     */
+    async canDisableMultisig() {
```

Comment:
IMO this should be made more specific like `isBelowMultisigActionsLimit` or similar. It provides more context to future readers since there's many reasons why multisig can't be disabled here.